sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/ApproximateQuantiles.java

```diff
@@ -358,6 +359,13 @@ private ApproximateQuantilesCombineFn(
         CoderRegistry registry, Coder<T> elementCoder) {
       return new QuantileStateCoder<>(compareFn, elementCoder);
     }
+
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("numQuantiles", numQuantiles)
+          .add("comparer", compareFn.getClass());
```

Comment:
Let's defer until this is an issue. In most cases comparators won't implement `toString()`, so the additional display data would be redundant and not as nicely formatted as `compareFn.getClass()`. I'd rather give an explicit was for comparators to register display data. 

Perhaps we could inspect `(compareFn instanceof HasDisplayData)`-- but, let's save that until we have a use-case.